subprojects/ide/src/main/java/org/gradle/plugins/ide/eclipse/model/ResourceFilterAppliesTo.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2011 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradle.plugins.ide.eclipse.model;
+
+/**
+ * TODO: documentation
+ *
+ * @since NEXT_VERSION
+ */
+public enum ResourceFilterAppliesTo {
```

Comment:
Would you like to sync the naming with the DSL?  I used ResourceFilterAppliesTo as "appliesTo" was used in the spec DSL.  Would you rather use "ResourceFilterTarget" and "target"?